src/theme/Footer/index.js

```diff
@@ -1,140 +1,24 @@
-/**
- * Copyright (c) Facebook, Inc. and its affiliates.
- *
- * This source code is licensed under the MIT license found in the
- * LICENSE file in the root directory of this source tree.
- */
+// Why is this swizzled?
+//   To fire Mixpanel events for logged-in users when the component renders.
+// Swizzled from version 2.0.0-rc.1.
+
+import React from "react";
+import Footer from "@theme-original/Footer";
+
 import { Auth0Provider, useAuth0 } from "@auth0/auth0-react";
 import BrowserOnly from "@docusaurus/BrowserOnly";
 import Head from "@docusaurus/Head";
-import isInternalUrl from "@docusaurus/isInternalUrl";
-import Link from "@docusaurus/Link";
-import { useThemeConfig } from "@docusaurus/theme-common";
-import useBaseUrl from "@docusaurus/useBaseUrl";
-import clsx from "clsx";
-import mixpanel from "mixpanel-browser";
-import React from "react";
-import IconExternalLink from "../IconExternalLink";
-import styles from "./styles.module.css";
 
-function FooterLink({ to, href, label, prependBaseUrlToHref, ...props }) {
```

Comment:
One nice thing about newer versions of docusaurus -- they made it easier to "wrap" theme components, which allowed us to delete a lot of this file.